src/vmspawn/vmspawn.c

```diff
@@ -104,6 +105,8 @@ static int help(void) {
                "     --linux=PATH           Specify the linux kernel for direct kernel boot\n"
                "     --initrd=PATH          Specify the initrd for direct kernel boot\n"
                "     --qemu-gui             Start QEMU in graphical mode\n"
+               "     --qemu-net=user|tap|none\n"
+               "                            Configure QEMU's networking stack\n"
```

Comment:
hmm, so i'd like to stay closer to nspawen's behaviour here.

hence maybe have `--network-tap` with a short `-n` here that is the simply one stop solution to proper networking. And then additionally (can be added later) an option `--network-tap-extra=` or so to add more of these.

And then `--network-user-mode` or so to get user mode networking. it's fine if --network-tap and --network-user-mode conflict, just fail if both are specified.